util-core/src/test/scala/com/twitter/util/TimeTest.scala

```diff
@@ -288,16 +288,11 @@ trait TimeLikeSpec[T <: TimeLike[T]] extends WordSpec with GeneratorDrivenProper
     }
   }
 
-  "floor" should {
-    "round down" in {
-      assert(fromSeconds(60).floor(1.minute) === fromSeconds(60))
-      assert(fromSeconds(100).floor(1.minute) === fromSeconds(60))
-      assert(fromSeconds(119).floor(1.minute) === fromSeconds(60))
-      assert(fromSeconds(120).floor(1.minute) === fromSeconds(120))
-    }
-
+  "rounding" should {
+    
     "maintain top and bottom" in {
       assert(Top.floor(1.hour) === Top)
+      assert(Bottom.floor(1.second) === Bottom)
```

Comment:
feel free to ignore:
for symmetry, keep this 1.hour too?  no reason for people who are next looking at this test to wonder, "why is one 1.hour and the other 1.second?" when, as far as I know, there isn't a reason.